### PR TITLE
Implement Mount Step

### PR DESCRIPTION
**Pull request by mkpankov**
Signed-off-by: Michael Pankov <work@michaelpankov.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2121)

### CODE REVIEW

File: iml-services/iml-action-runner/src/local_actions.rs [hunk 1]
```diff
@@ -119,11 +120,19 @@ pub async fn handle_local_action(
                 spawn_plugin(fut, in_flight, id);
 
                 rx.await.map_err(ActionRunnerError::OneShotCanceledError)
-            } else {
-                Err(ActionRunnerError::RequiredError(error::RequiredError(
-                    format!("Could not find action {} in local registry", action),
-                )))
             }
-        }
+            "mount_snapshot" => {
```
Comment (ip1981) [inline iml-services/iml-action-runner/src/local_actions.rs:124]: For consistency (with iml-agent, lctl, etc.) it better be `snapshot_mount`.

File: iml-services/iml-action-runner/src/local_actions.rs [hunk 2]
```diff
@@ -119,11 +121,23 @@ pub async fn handle_local_action(
...
-                    format!("Could not find action {} in local registry", action),
-                )))
             }
-        }
+            "snapshot_mount" | "snapshot_unmount" => {
+                let rx = add_in_flight(Arc::clone(&in_flight), id.clone()).await;
+                let args2 = args.clone();
+
+                let fut = wrap_plugin(args, move |fqdn: Fqdn| {
+                    let (_, fut) = invoke_rust_agent(fqdn, action, args2);
```
Comment (jgrund) [inline iml-services/iml-action-runner/src/local_actions.rs:130]: This will ignore cancellation, which needs to be handled.

Comment (mkpankov) [reply iml-services/iml-action-runner/src/local_actions.rs]: Done

File: chroma_core/models/host.py [hunk 3]
```diff
@@ -1783,3 +1783,13 @@ def alert_message(self):
 
 class NoNidsPresent(Exception):
     pass
+
+
+class MountSnapshotStep(Step):
+    def run(self, kwargs):
+        self.invoke_rust_agent_expect_result(kwargs["host"], "snapshot_mount", [kwargs["fsname"], kwargs["name"]])
```
Comment (nlinker) [inline chroma_core/models/host.py:1790]: Btw, why `kwargs['host']` was choosen to be the first parameter?

Comment (mkpankov) [reply chroma_core/models/host.py:1790]: That's the syntax of `invoke_rust_agent_expect_result` - `host` is first

Comment (jgrund) [reply chroma_core/models/host.py:1790]: The first parameter needs to be the fqdn of the host to send the RPC to.

**Pull request merged**